src/dataset-builder/ConceptSearch.ts

```diff
@@ -21,8 +21,9 @@ type ConceptSearchProps = {
   readonly onCommit: (selected: Concept[]) => void;
   readonly onOpenHierarchy: (
     domainOption: SnapshotBuilderDomainOption,
-    selected: Concept[],
-    searchText: string
+    cart: Concept[],
+    searchText: string,
+    selectedConcept?: Concept
```

Comment:
I think the name selectedConcept is a bit ambiguous as we also use selected to mean in the cart. Maybe this could be called openedConcept?

Reply:
I agree with this!